pkg/cluster/util_test.go

```diff
@@ -650,7 +650,7 @@ func Test_trimCronjobName(t *testing.T) {
 	}
 }
 
-func TestisInMaintenanceWindow(t *testing.T) {
```

Comment:
my local go was complaining and failing unit test because second word after `Test` did not have a capital letter.